### ISSUE TITLE
Add metrics for endpoints

### ISSUE THREAD

**Issue by arianvp** [dstotijn/ct-diag-server#4]
This can be easily added using `promhttp` library https://prometheus.io/docs/guides/go-application/#how-go-exposition-works

**Comment by dstotijn** [comment]
@arianvp Yes, good idea! Were you planning to do a PR yourself? If so, please assign to yourself. Otherwise I'm also happy to do it.
